src/UI/Activity/Queue/QueueLayout.js

```diff
@@ -1,7 +1,7 @@
 var Marionette = require('marionette');
 var Backgrid = require('backgrid');
 var QueueCollection = require('./QueueCollection');
-var SeriesTitleCell = require('../../Cells/MovieTitleCell');
+var MovieTitleCell = require('../../Cells/MovieTitleCell');
```

Comment:
Ah ok, in that case good to merge. (Just thought you ment to use that one as you changed that one.)